home/app_manager.py

```diff
@@ -90,6 +92,24 @@ class AppManagerWidget(ipw.VBox):
         </div>"""
     )
 
+    CORE_DEPENDENCIES_COMPATIBILTIY_WARNING = Template(
+        """<div class="alert alert-danger">
+    The version to be install of this app is not compatible with this AiiDAlab environment.
+    </div>"""
+    )
+
+    DEPENDENCIES_INSTALL_INFO = Template(
+        """<div class="alert alert-info alert-dismissible">
+        <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+            Following dependencie(s) will be changed:
```

Comment:
```suggestion
            The following dependencie(s) will be changed:
```

Reply:
"dependecie" is not a word so putting (s) in parentheses looks super weird to me. I'd just go with `dependencies` even if there might be just one.

```suggestion
            The following dependencies will be changed:
```